docs/reference/latency.md

```diff
@@ -0,0 +1,96 @@
+You start a transaction by submitting a formatted request to the `/send` endpoint on a node.
+The code provided by the `/send` endpoint performs some simple initial checks that validate the transaction has a reasonably high chance of being executed successfully. 
+These initial checks ensure that you learn if a transaction is invalid as soon as possible. 
+If the transaction passes the initial checks, the `/send` process adds the transaction to the mempool for the local node. 
+The transaction is then gossiped to remote nodes, and, eventually, one of the nodes is selected to mine the next block. 
+After the mined block is added to its target chain, the block with the transaction result is synchronized on the local node, and a request to the /poll endpoint returns the transaction result.
+
+Under normal load, the initial checks should take less than a millisecond for a single transaction.
+However, if a node in under heavy load—for example, because the PactService is contending with a high volume of `/local` or `/send` calls or because the node is catching up—you might experience delays in transaction processing. 
+It's worth noting that if a node is too busy, it will always prioritize staying caught up over sending transactions.
+
+The propagation of transactions in the mempool to other nodes takes, on average, 10 seconds, but it can take longer if there are networking delays, such as increased network traffic or slow network connections between nodes.
+Block propagation back to the local node is usually a matter of seconds.
+
+## Cross-chain transfers
+
+As discussed in [Transaction format and flow](/smart-contracts/transactions), a cross-chain transfer is essentially two separate but related transactions.
+The time it takes to complete a cross-chain transfer can depend on several factors, including the physical distance between the chain nodes in the network, network congestion, and the synchronization of state between the chains involved.
+
+For two distant chains—representing the worst case—a cross-chain transfer is expected to take the following time to complete:
+
+- The initial transaction is a exec transaction with an average time from submission to completion of 45 seconds.
+- Two or more blocks must be propagated to the target chain so that the source and target chain share the same view of state, with an average time of 30 seconds per block.
+- The continuation step is a second transaction, again with an average time of 45 seconds.
+
+Therefore, a cross-chain transfer typically takes 2 minutes and 30 seconds to complete.
+If either the source or target chain is ahead of the other chain, more blocks might be required to synchronize the chains before the transaction can complete, in which case the transfer might take additional time.
+
+## Mining per block
+
+Mining a block takes 16 seconds on average.
```

Comment:
```suggestion
Mining a block takes 16 seconds on average with the current 20-chain graph. This will decrease as more chains are added and chains spend more time blocked from creating new blocks.
```

Reply:
I just want to qualify this for the future.

Reply:
Just want to clarify what "this" refers to and that _decrease_ is correct in this context:

> The average time it takes to mine a block will decrease as more chains are added and chains spend more time blocked from creating new blocks.